charts/spire/charts/spire-server/README.md

```diff
@@ -108,8 +108,10 @@ A Helm chart to install the SPIRE server.
 | nodeSelector | object | `{}` | Select specific nodes to run on (currently only amd64 is supported by Tornjak) |
 | notifier.k8sbundle.namespace | string | `""` | Namespace to push the bundle into, if blank will default to SPIRE Server namespace |
 | persistence.accessMode | string | `"ReadWriteOnce"` |  |
+| persistence.hostPath | string | `""` | Which path to use on the host when type = hostPath |
 | persistence.size | string | `"1Gi"` |  |
 | persistence.storageClass | string | `nil` |  |
+| persistence.type | string | `"pvc"` | What type of volume to use for persistence. Valid options pvc (recommended), hostPath, emptyDir (testing only) |
```

Comment:
Same comment as elsewhere that this should default to emptyDir.

Reply:
Why should it default to a value that is for testing only and not the recommended value?

Reply:
I'm willing to not take a stand on this item.  I'll let it be